pom.xml

```diff
@@ -45,10 +45,10 @@
     <version.io.opentracing.contrib-opentracing-tracerresolver>0.1.5</version.io.opentracing.contrib-opentracing-tracerresolver>
     <version.io.opentracing.contrib-opentracing-spring-tracer-configuration-starter>0.1.0</version.io.opentracing.contrib-opentracing-spring-tracer-configuration-starter>
     <version.junit>4.12</version.junit>
-    <version.org.mockito-mockito-core>1.10.19</version.org.mockito-mockito-core>
-    <version.org.springframework.boot>1.5.12.RELEASE</version.org.springframework.boot>
+    <version.org.mockito-mockito-core>2.23.4</version.org.mockito-mockito-core>
+    <version.org.springframework.boot>2.1.0.RELEASE</version.org.springframework.boot>
     <!-- Should match version from https://github.com/spring-projects/spring-boot/blob/1.5.x/spring-boot-dependencies/pom.xml  -->
```

Comment:
could you please also change this link poiting to SB 2.x?  https://github.com/spring-projects/spring-boot/blob/master/spring-boot-project/spring-boot-dependencies/pom.xml

Reply:
Good catch!